libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -131,6 +133,22 @@ interface MapboxNativeNavigator {
      */
     fun getHistoryRecorderHandle(): HistoryRecorderHandle?
 
+    /**
+     * Returns the native class that allows the sdk to record native history files.
+     *
+     * @return null when there is no directory to write files,
+     *     or when the handle did not [HistoryRecorderHandle.build]
+     */
+    fun getCopilotHistoryRecorderHandle(): HistoryRecorderHandle?
+
+    /**
+     * Returns the native class that allows the sdk to record native history files.
+     *
+     * @return null when there is no directory to write files,
+     *     or when the handle did not [HistoryRecorderHandle.build]
+     */
+    fun getCompositeHistoryRecorderHandle(): HistoryRecorderHandle?
```

Comment:
What's the difference between this method and the method above? Docs are the same...